src/main/java/com/feedzai/commons/sql/abstraction/dml/UpdateFrom.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2020 Feedzai
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.feedzai.commons.sql.abstraction.dml;
+
+/**
+ * PDB Update From Operator.
+ *
+ * @author Francisco Santos (francisco.santos@feedzai.com)
+ * @since @@@feedzai.next.release@@@
+ */
+public class UpdateFrom extends Update {
+
+    /**
+     * The from clause.
+     */
+    private final Expression from;
+
+    /**
+     * Creates a new {@link UpdateFrom} instance.
+     *
+     * @param table the table to update.
+     * @param from the from.
+     * @implNote Apart from PostgreSQL, engines do not support this operator, so the final SQL will be different.
```

Comment:
If this is true:
- where is the translator implementation for DB2?
- why is PostgreSQL/Cockroach implementation the default (implemented in AbstractTranslator), and not the code used for H2, MySQL and Oracle, which is exactly the same?

Reply:
Well caught!
DB2 also supports it, and yes the specific implementation should be PostgreSQL and DB2 not the other way around  😄